lms/djangoapps/courseware/tests/test_footer.py

```diff
@@ -36,26 +37,26 @@ class TestFooter(TestCase):
         "youtube": "https://www.youtube.com/"
     }
 
+    @with_is_edx_domain(True)
```

Comment:
Yes, eventually this repo would only have tests like, "does theming make this footer change?" and all the edx.org specific details would be in a separate theme.  One of the goals of this theming feature is to be comprehensive enough that all of the edx.org look-and-feel differences are in the theme.